### PR TITLE
Add nitrotype.com

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Nitro Type is an online multiplayer typing game that helps users improve their typing skills through competitive and engaging races. The platform provides 2FA for the account login verification process. The website is ranked 6,508 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=nitrotype.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: entries/n/nitrotype.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Nitro Type": {
+    "domain": "nitrotype.com",
+    "img": "nitrotype.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.nitrotype.com/news/read/254/secure-your-nitro-account-with-two-factor-authentication-2fa",
```
Comment (Carlgo11) [inline entries/n/nitrotype.com.json]:
```suggestion
    "documentation": "https://www.nitrotype.com/news/read/254/-",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Here's an SVG icon you can use:
icon

**Comment by Yang-Nankai** [comment]
@Carlgo11 Thanks! I have revised the question you asked and committed again. 
p.s. I would like to ask where you found the SVG you provided, I tried to use a search engine to find it but I failed. ✋

**Comment by Carlgo11** [comment]
@Yang-Nankai I just copied the N from the logo on their website.
image

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**